recipes/multiqc/meta.yaml

```diff
@@ -116,8 +117,7 @@ test:
 
   commands:
     # click requires a unicode locale
-    - LC_ALL=C.UTF-8 LANG=C.UTF-8 multiqc --version # [linux]
-    - LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 multiqc --version # [osx]
+    - LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 multiqc --version
```

Comment:
You can even just use
```yaml
    - multiqc --version
```
since the `en_US.UTF-8` locale is already set in the extended base image.

Reply:
True